.github/workflows/tests.yml

```diff
@@ -21,24 +22,42 @@ jobs:
             experimental: false
             coverage: true
 
+          - os: "windows-latest"
+            python-version: ""
+            experimental: false
+
     steps:
       - uses: actions/checkout@v3
 
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+        if: matrix.os == 'ubuntu-latest'
+
+      - name: Set up micromamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: "./environment.yml"
+          init-shell: >-
+            powershell
+        if: matrix.os == 'windows-latest'
 
-      - name: Install dependencies
+      - name: Install Linux dependencies
         run: |
           sudo apt install libgeos-dev

```

Comment:
Why we don't use micromamba on linux systems to install cartopy?

Reply:
It seems much slower to setup the micromamba environment and install cartopy in it (at least on windows) compared to have it built from `pip install`. I can test how it goes on linux if you want